components/ws-manager/pkg/manager/create.go

```diff
@@ -212,6 +213,38 @@ func mergeProbe(dst, src reflect.Value) (err error) {
 	return nil
 }
 
+func (m *Manager) createPVCForWorkspacePod(startContext *startWorkspaceContext) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
We'll want some basic test coverage of this. Introducing a fixture test akin to the [CDWP tests](https://github.com/gitpod-io/gitpod/blob/dcf82f628eac7c380d0677b5c3f4485e293ee560/components/ws-manager/pkg/manager/create_test.go#L22) should be straight forward.

(doesn't necessarily have to be in this PR)

Reply:
:100: 
I think once everything is working (and code is no longer changing with every commit that I make :sweat_smile: ), I will add all missing tests (to make sure I don't forget I added a sub section into epic regarding this)